ui/pages/Project/Project.jsx

```diff
@@ -45,6 +46,7 @@ class Project extends React.PureComponent {
           <Route path={`${match.url}/analysis_group/:analysisGroupGuid`} component={ProjectPageUI} />
           <Route path={`${match.url}/family_page/:familyGuid/matchmaker_exchange`} component={Matchmaker} />
           <Route path={`${match.url}/family_page/:familyGuid`} component={FamilyPage} />
+          <Route path={`${match.url}/rnaseq_results/:familyGuid/:individualGuid`} component={RnaSeqResultPage} />
```

Comment:
We actually want this route nested under the family page. So this file should not change at all, but the `FamilyPage` should actually be structured very similarly to this file, where it loads the family data and then renders a `Switch` where `match.url` maps to the `FamilyDetail` and `${match.url}/rna/:individualGuid` routes to the new rna page.

Reply:
Good points. I will try.

Reply:
I did some experiments and found the page open with <Link ... targe="_blank" > doesn't have access to its parent window's state. Maybe, it is reasonable for the `get_individual_rna_seq_data` API to fetch the minimum required states, including the family and individual info.

Reply:
As I have stated multiple times in this review, this needs to be a nested Route. It is nested under the family page Route, which loads the family data. Then, after all the family specific data is loaded, it renders the rna-specific Route and loads the RNA specific data, but will not need to load any family specific data as that is already loaded. This change. WILL fix the issue you are having and is the way we want to handle fetching page-specific data here. It is basically the exact same as the way we handle all the project-specific pages, by first loading the project data and only then loading subsequent page specific data, so it is consistent with our other architecture

Reply:
The `Switch` and `Route` are added to the `FamilyPage`. The `state` works now.